### ISSUE TITLE
Visitor Profile | User ID export format not giving other options

### ISSUE THREAD

**Issue by enual** [piwik/piwik#21588]
### What happened?

When you hover your mouse to export User ID, a message will appear "Export this dataset in other formats". When you click it, it will open a new tab for xml file and will not give you any other option:

![image](https://github.com/matomo-org/matomo/assets/132400623/aa15e5d2-00de-4b87-85be-482896e741b0)

### What should happen?

As per message "Export this dataset in other formats", it should give them other file options to download.

### How can this be reproduced?

From Matomo Dashboard, go to Visitors > User IDs > click the User

### Matomo major version

Matomo 3

### Matomo minor or patch Version

na

### PHP version

na

### Server operating system

na

### Computer operating system

Windows

### Validations

- [X] Read our [Contributing Guidelines](https://github.com/matomo-org/matomo/blob/5.x-dev/CONTRIBUTING.md).
- [X] Follow our [Security Policy](https://github.com/matomo-org/matomo/blob/5.x-dev/SECURITY.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating duplicates.
- [X] The provided steps to reproduce is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the Bug.

**Comment by michalkleiner** [comment]
Yes, the message might be misleading in that there might be other formats whereas at the moment it's just an XML format.
@enual is the user reporting this asking for any specific format or is the issue just the message not be accurate?

**Comment by enual** [comment]
@michalkleiner they didn't ask for a specific format, they just notice that it's not giving them another options.

**Comment by sgiehl** [comment]
With a couple of adjustments it might be possible to use our report export overlay for that case as well. That would allow to also export to other formats. But the most options in the overlay might not make sense in that case and should be hidden I guess.

**Comment by enual** [comment]
That's a good idea. Thanks for letting me know.
